packages/reload/reload.js

```diff
@@ -223,7 +223,7 @@ Reload._reload = function (options) {
       // with the server if we still have a valid cached copy. This doesn't work
       // when the location contains a hash however, because that wouldn't reload
       // the page and just scroll to the hash location instead.
-      if (window.location.hash) {
+      if (window.location.hash || _.last(window.location.href) === '#') {
```

Comment:
Sure thing. Just out of curiosity, what's wrong with using underscore? It's used in this module already. I didn't really understand CircleCI's output - couldn't grok how it's linked to underscore.